opendc-kernel-omega/src/main/kotlin/com/atlarge/opendc/omega/OmegaSimulation.kt

```diff
@@ -430,11 +430,6 @@ internal class OmegaSimulation<M>(bootstrap: Bootstrap<M>) : Simulation<M>, Boot
          *
          * @param exception The exception to resume with.
          */
-        override fun resumeWithException(exception: Throwable) {
-            // Deregister process from registry in order to have the GC collect this context:w
-            registry.remove(process)
-
-            logger.error(exception) { "An exception occurred during the execution of a process" }
-        }
+        override fun resumeWithException(exception: Throwable) = throw exception
```

Comment:
Is this then still a correct name for the function, seeing as it doesn't `resume` anymore?

Reply:
It is part of the [Continuation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.coroutines.experimental/-continuation/index.html) interface so we can't change the name. I see it as resuming by propagating the exception to the caller (`step()` in this case) :)

Reply:
Hmm ok, makes sense :)